### DISCUSSION TITLE
ΓΕ1: Περσόνες, σενάρια και σχετικές εφαρμογές

### DISCUSSION THREAD

**Discussion by VasilisKokkalis** [sdy60/syllabus#163] [Q&A] [answered]
Καλησπέρα σας, Θα ήθελα να ρωτήσω σχετικά με το 2ο ερώτημα στο οποίο  αναφέρεται να παρουσιάσουμε μια περσόνα και ένα σενάριο χρήσης της εφαρμογής μας. Πρόκειται για τις σχετικές εφαρμογές που αναπτύξαμε στο ερώτημα 1 του Β μέρους ή μια φανταστική δική μας;

**Comment by VagNta** [comment, marked as answer]
Καλημέρα.
Πιστεύω πως είναι συνδυασμός και των δυο, γιατί εκεί θα βασιστούμε από τι στιγμή που στο επόμενο ερώτημα αναφέρει πως: Για το **σενάριο/περσόνα που αναπτύξατε παραπάνω σχεδιάστε ενδεικτικές οθόνες**
